ee/secureenclavesigner/secureenclavesigner.go

```diff
@@ -0,0 +1,237 @@
+//go:build darwin
+// +build darwin
+
+package secureenclavesigner
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/kolide/krypto/pkg/echelper"
+	"github.com/kolide/launcher/ee/allowedcmd"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+const (
+	CreateKeyCmd = "create-key"
+	SignCmd      = "sign"
+)
+
+type SignRequest struct {
+	SecureEnclaveRequest
+	// SecureEnclavePubKey is the B64 encoded DER of the public key to be used to verify the signature
```

Comment:
Isn't it the key that should be used to sign the request?

Reply:
good catch